Docs: Add Video Onboarding Article (#904)
telerik/blazor-docs: getting-started/video-onboarding.md

---
title: Video Onboarding
page_title: Video Onboarding
description: "Get started with Telerik UI for Blazor by following a step-by-step video tutorial that helps you develop a real-life project."
slug: getting-started/video-onboarding
previous_url: /online-technical-training
tags: get,started,video,onboarding,classroom,course,learning
published: true
position: 50
---

# Video Onboarding

> Telerik UI for Blazor has its own [video getting started course](https://learn.telerik.com/learn/course/internal/view/elearning/27/telerik-ui-for-blazor) available to both active trial users and active license holders in [Virtual Classroom](https://learn.telerik.com/learn). The training course is developed to help you get started with the Telerik UI for Blazor components and features. It aims to put you in the shoes of an engineer who adds new features to an existing application.

Reviewer: ```suggestion
Active license holders and trial users have access to [Telerik UI for Blazor getting started video course](https://learn.telerik.com/learn/course/internal/view/elearning/27/telerik-ui-for-blazor) available in [Virtual Classroom](https://learn.telerik.com/learn). The training course is developed to help you get started with the Telerik UI for Blazor components and features. It aims to put you in the shoes of an engineer who adds new features to an existing application.
```

Author: Updated

The course starts with an introduction to the Blazor framework: the Server and WebAssembly hosting models, Blazor components basics&mdash;life cycle, data binding, different types of project creation, and using code-behind approach to component creation.

The training flow then takes you to an introduction to Telerik UI for Blazor, the ways to install and setup the components. Further, you get into creating your RPS issue tracker Blazor application, enhancing it with one of the built-in themes and basic components like Button. Then, you use more advanced components, such as Calendar, Grid, and Chart. You will also dive deep into the advanced Grid features, such as filtering and exporting data to Excel.

> You will also learn about other important Telerik UI for Blazor built-in features, such as Globalization and Localization, Accessibility and Keyboard Navigation, as well as options for Document Processing and Reporting. The final module is dedicated to deployment and will show you how to deploy both Blazor Server and Wasm applications to Azure.

Reviewer: Maybe a list? :)

Author: It would fit perfectly in a list but I wanted to keep it more like a story.
